### PR TITLE
Quest

### PR DESCRIPTION
**Pull request by tul1**

### CODE REVIEW

File: src/datamodel/quest/quest_proposer.rs [hunk 1]
```diff
@@ -0,0 +1,11 @@
+use crate::datamodel::quest::quest_proposer_role_action_manager::QuestProposerRoleActionManager;
+
+pub struct QuestPorposer {
```
Comment (manuelaguirre) [inline src/datamodel/quest/quest_proposer.rs]: typo here

Comment (tul1) [reply src/datamodel/quest/quest_proposer.rs]: done

File: src/datamodel/quest/quest.rs [hunk 2]
```diff
@@ -0,0 +1,9 @@
+use crate::datamodel::quest::quest_objective::QuestObjective;
+use crate::datamodel::quest::quest_proposer::QuestProposer;
+use crate::datamodel::quest::quest_member::QuestMember;
+
+pub struct Quest{
```
Comment (julianmateu) [inline src/datamodel/quest/quest.rs]:
```suggestion
pub struct Quest {
```

File: src/datamodel/quest/finish_quest.rs [hunk 3]
```diff
@@ -0,0 +1,6 @@
+use crate::datamodel::quest::Quest;
+
+pub struct FinishQuest {
```
Comment (julianmateu) [inline src/datamodel/quest/finish_quest.rs]:
```suggestion
pub struct FinishedQuest {
```

File: src/datamodel/quest/finish_quest.rs [hunk 4]
```diff
@@ -0,0 +1,6 @@
...
+    won: bool,
+    quest: Quest,
+}
```
Comment (julianmateu) [inline src/datamodel/quest/finish_quest.rs]: The file should be renamed to `finished_quest.rs` too

File: src/datamodel/quest/mod.rs [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+pub mod finish_quest;
+
+
+pub use quest::Quest;
+pub use quest_objective::QuestObjective;
+pub use quest_member_role_action_manager::QuestMemberRoleActionManager;
+pub use quest_member::QuestMember;
+pub use quest_proposer_role_action_manager::QuestProposerRoleActionManager;
+pub use quest_proposer::QuestProposer;
+pub use finish_quest::FinishQuest;
```
Comment (julianmateu) [inline src/datamodel/quest/mod.rs]:
```suggestion
pub use finished_quest::FinishedQuest;
```

**Review by julianmateu — CHANGES_REQUESTED** [review]

**Review by julianmateu — APPROVED** [review]

**Pull request merged**